apps/web/tests/utils.ts/browser.ts

```diff
@@ -0,0 +1,31 @@
+import { BrowserContext, Locator, Page } from '@playwright/test';
+
+import { getSession, ISessionOptions } from './plugins';
+
+export async function initializeSession(context: BrowserContext, settings: ISessionOptions = {}) {
+  const session = await getSession(settings);
+
+  await context.addInitScript((session) => {
+    (window as any).isPlaywright = true;
+    localStorage.setItem('auth_token', session.token);
+  }, session);
+
+  return session;
+}
+
+export function getByTestId(page: Page | Locator, selector: string, options?: Parameters<Page['locator']>[1]) {
+  return page.locator(`[data-test-id="${selector}"]`, options);
+}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏I wonder, can we overwrite the built in page.getByTestId to work with our customer selector?

Reply:
I think one of the things that made the migration header was our plugin methods that we overrided, I wonder if using a builder function will be a better solution so we won't use strings and not couple ourselves to playwright at the same time, so if we would want to migrate any test in the future it would be easier to do.

Reply:
with fixtures again :)